CAM-9852 (#2626)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

  Besides the user interface, you can automatically schedule instances using the instances API with a [POST or PUT](https://www.ctl.io/api-docs/cam/#application-lifecycle-management-instances-api) request.

### Protecting Instance Shutdown/Termination

Avoid accidental operations on particular instances. *Instance Protection Flag* allows to protect an instance for manual **shutdown** or **termination**. Instance still being affected by *scheduled termination or shutdown*. 

> *Instance Protection Flag* forbid operators to run shutdown/terminate operations on API/UI level, also affecting at **provider level** if implements this feature. 

Reviewer: ```suggestion
*Instance Protection Flag* prevents the user to perform shutdown/terminate operations through the user interface or an API call. It can also be enabled at **provider level**, affecting all instances deployed on it. 
```

> *Instance Protection Flag* can be setted on **Deployment Policy Level** and on **Instance Deployment Level**. 

Reviewer: ```suggestion
*Instance Protection Flag* can be set at **Deployment Policy Level** or **Instance Level**. 
```

> Follow these steps to enable instance protection on **deployment policy**:

Reviewer: ```suggestion
Follow these steps to enable instance protection on any **deployment policy** box:
```

1. From the boxes page, click **New**
2. Select **Deployment Policy**

> 3. In the New Deployment Policy Box dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggle under **expiration** dropdown.

Reviewer: ```suggestion
3. In the New or Edit Deployment Policy Box dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggle under **expiration** dropdown.
```

4. When done, click **Save**

> **Note:** Instances affected by created policy box will propagate the protection configuration flags, preventing operators to disable them.

Reviewer: ```suggestion
**Note:** Instances deployed through this policy box will inherit the protection configuration flags, preventing the user to disable them.
```

> Follow these steps to protect an instance on **deployment**:

Reviewer: ```suggestion
Follow these steps to protect an instance at **deployment** time:
```
You should mention that for a deployed instance you will be able to change these toggles by editing its details

**Steps**
1. From the instances page, click **New**.
2. Select a box you want to deploy.
3. Select a deployment policy box from availables in policy dropdown. 
4. In the New Instance dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggles under **Expiration** dropdown.
5. When done, click **Deploy**

> **Note:** If policy box have enabled **Manual Shutdown Protection** or **Manual Terminate Protection** it will propagate to the current instance

Reviewer: ```suggestion
**Note:** If policy box used for deployment has **Manual Shutdown Protection** or **Manual Terminate Protection** enabled, it will be inherited by the current instance, preventing the user to disable them.
```

deployment dialog, disabling related toggles in current dialog.

> #### Registering Instances

Reviewer: ```suggestion
#### Instance protection on registered Instances
```

  Besides the user interface, you can automatically schedule instances using the instances API with a [POST or PUT](https://www.ctl.io/api-docs/cam/#application-lifecycle-management-instances-api) request.

### Protecting Instance Shutdown/Termination

> Avoid accidental operations on particular instances. *Instance Protection Flag* allows to protect an instance for manual **shutdown** or **termination**. Instance still being affected by *scheduled termination or shutdown*. 

Reviewer: Sorry, but now that I read it again, I have another suggestion:
```suggestion
Avoid accidental operations on particular instances. *Instance Protection Flag* allows to protect an instance for manual **shutdown** or **termination**. The instance will still be affected by *scheduled termination or shutdown* configured values if any. 
```

*Instance Protection feature* prevents the user to perform shutdown/terminate operations on API/UI level, also affecting at **provider level** if implements this feature. 

*Instance Protection feature* can be set at **Deployment Policy Level** or **Instance Deployment Level**. 

Follow these steps to enable instance protection on any **deployment policy** box:

1. From the boxes page, click **New**
2. Select **Deployment Policy**

> ![Deployment Policy Box Dialog](../../images/cloud-application-manager/protecting-instance-deployment-policy-1.png)

Reviewer 2: image file is missing or path is wrong

3. In the New or Edit Deployment Policy Box dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggle under **expiration** dropdown.
4. When done, click **Save**

**Note:** Instances deployed through this box will inherit the protection configuration flags, preventing the user to disable them.

> ![Instance Deployment (Inherit) Dialog](../../images/cloud-application-manager/protecting-instance-deployment-inherited-2.png)

Reviewer 2: image file is missing or path is wrong

Follow these steps to protect an instance at **deployment** time:

**Steps**
1. From the instances page, click **New**.
2. Select a box you want to deploy.
3. Select a deployment policy box from availables in policy dropdown. 
4. In the New Instance dialog, enable the **Manual Shutdown Protection** or **Manual Terminate Protection** toggles under **Expiration** dropdown.

> ![Deployment Policy Box Dialog](../../images/cloud-application-manager/protecting-instance-deployment-policy-3.png)

Reviewer 2: image file is missing or path is wrong
